src/Microsoft.Diagnostics.EventFlow.Core/Implementations/Metadata/DataRetrievalResult.cs

```diff
@@ -12,24 +13,20 @@ public enum DataRetrievalStatus
         Success = 0,
         MetadataPropertyMissing = 1,
         InvalidMetadataValue = 2,
-        DataMissingOrInvalid = 3
+        DataMissingOrInvalid = 3,
+        InvalidMetadataType = 4
     }
 
     public class DataRetrievalResult
     {
-        private static readonly DataRetrievalResult SuccessResult = new DataRetrievalResult() { Status = DataRetrievalStatus.Success };
+        public static readonly DataRetrievalResult Success = new DataRetrievalResult() { Status = DataRetrievalStatus.Success };
 
         public DataRetrievalStatus Status { get; private set; }
         public string Message { get; private set; }
 
         // Prevent public construction
         private DataRetrievalResult() { }
 
-        public static DataRetrievalResult Success()
```

Comment:
For the Success property there is simply no good reason to use a method. Other static methods in this class are factory methods that need parameters. I do not think this is particularly confusing.